shared/src/main/scala/com/cibo/evilplot/colors/GradientUtils.scala

```diff
@@ -67,6 +67,7 @@ object GradientUtils {
         {
           if (d < min) colors.head
           else if (d > max) colors.last
+          else if (d.isNaN) RGBA(0,0,0,0)
```

Comment:
Given that RGBA is currently going to be translated to HSLA, it will probably be faster, with no loss in clarity, to make this an HSLA(0,0,0,0), or go with Clear, which is ultimately the same thing, except as a separate implementation of Color for some reason.

Reply:
Ah, that's much cleaner. Thanks!